include/experimental/__p0009_bits/mdspan.hpp

```diff
@@ -219,6 +222,68 @@ class mdspan
   //--------------------------------------------------------------------------------
   // [mdspan.basic.mapping], mdspan mapping domain multidimensional index to access codomain element
 
+  MDSPAN_TEMPLATE_REQUIRES(
+    class... SizeTypes,
+    /* requires */ (
+      _MDSPAN_FOLD_AND(_MDSPAN_TRAIT(std::is_convertible, SizeTypes, index_type) /* && ... */) &&
+      _MDSPAN_FOLD_AND(_MDSPAN_TRAIT(std::is_nothrow_constructible, index_type, SizeTypes) /* && ... */) &&
+      (rank() == sizeof...(SizeTypes))
+    )
+  )
+  constexpr reference at(SizeTypes... indices) const
+  {
+    size_t r = 0;
+    for (const auto& index : {indices...}) {
+      if (index >= __mapping_ref().extents().extent(r) || index < 0) {
```

Comment:
Ugh sorry to complicate things further -- but I'm pretty sure some compilers will warn that this is a pointless comparison when index is size_t, so you (may) need to check for that. Also looks like you need to make sure the 0 is of the correct type